xdsl/dialects/builtin.py

```diff
@@ -1248,6 +1248,13 @@ def __len__(self) -> int:
         return len(self.data.data) // self.elt_type.size
 
 
+DenseI64ArrayConstraint = ParamAttrConstraint(DenseArrayBase, [i64, BytesAttr])
+"""Type constraint for DenseArrays containing integers of i64 integers."""
+
+DenseI64ArrayType: TypeAlias = Annotated[DenseArrayBase, DenseI64ArrayConstraint]
```

Comment:
```suggestion
DenseI64Array: TypeAlias = Annotated[DenseArrayBase, DenseI64ArrayConstraint]
```

Reply:
But also this seems not to be used?

Reply:
We do this with other types, so I wasn't sure.
Delete?